src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1020,6 +1021,11 @@ protected DynamoModel(IStartConfiguration config)
             DynamoReady(new ReadyParams(this));
         }
 
+        private void AuthServicesEvents_AuthProviderRequested(RequstAuthProviderEventArgs args)
+        {
+            args.FoundAuthProvider = AuthenticationManager.AuthProvider;
```

Comment:
The name of this property sounds like it's a boolean. Why not just name it `AuthProvider`?

Reply:
thanks @aparajit-pratap - I can change the name and make it nullable.

Reply:
ugh, actually I can't easily do this because netstd2 is stuck on c# 7.3 by default - I will just change the name and document it.